### PR TITLE
Action page fixes

### PR DESCRIPTION
**Pull request by pariparajuli**
Patch up edits w/ css changes and internal error fixes

### CODE REVIEW

**Comment by pariparajuli** [comment]
Fixes for https://github.com/buildbuddy-io/buildbuddy-internal/issues/779
Screen Shot 2021-08-05 at 4 36 00 PM
Screen Shot 2021-08-05 at 4 36 36 PM

**Review by siggisim — APPROVED** [review]
Thanks for fixing these!

**Review by bduffany — APPROVED** [review]
Nice!

File: app/invocation/invocation_action_card.tsx [hunk 1]
```diff
@@ -45,7 +44,7 @@ export default class InvocationActionCardComponent extends React.Component<Props
         this.fetchCommand(action);
         this.fetchInputRoot(action.inputRootDigest);
       })
-      .catch((e) => errorService.handleError(e));
+      .catch((e) => console.log(e));
```
[the remark this answers is not in the document]
Comment (pariparajuli) [reply app/invocation/invocation_action_card.tsx:47]: Does handing it off to the errorService create a popup on the UI? I don't know if we want a pop up. @siggisim thoughts?

**Pull request merged**